test/poolFixtures.hpp

```diff
@@ -82,13 +82,16 @@ struct umfPoolTest : umf_test::test,
         test::SetUp();
 
         pool = poolCreateExtUnique(this->GetParam());
+
```

Comment:
What about making it as a member of `struct umfPoolTest`?

Reply:
`ncores` was removed